### ISSUE TITLE
Terms and Conditions and Data Privacy statement

### ISSUE THREAD

**Issue by nhrishi** [PharmaLedger-IMI/epi-workspace#126]
The Lightweight App and Native Mobile apps should display the privacy statement and Terms of use. Below links should be embedded. 

**Privacy Policy**
URL: [https://app.termly.io/document/privacy-policy/40430c0a-9e30-4690-b4a7-2c2f56433919](https://urldefense.com/v3/__https://app.termly.io/document/privacy-policy/40430c0a-9e30-4690-b4a7-2c2f56433919__;!!N3hqHg43uw!vlzinluv-33KHvobGc7SklLFAa05KN4mKVs23dcof7l5d7kRg7osS3JGx-kCTZe64JHCWc21ot1Ys7GNEuVXOihGzYxI03a5$) 
DIV: 
```
<div name="termly-embed" data-id="40430c0a-9e30-4690-b4a7-2c2f56433919" data-type="iframe"></div>
<script type="text/javascript">(function(d, s, id) {
  var js, tjs = d.getElementsByTagName(s)[0];
  if (d.getElementById(id)) return;
  js = d.createElement(s); [js.id](https://urldefense.com/v3/__http://js.id__;!!N3hqHg43uw!vlzinluv-33KHvobGc7SklLFAa05KN4mKVs23dcof7l5d7kRg7osS3JGx-kCTZe64JHCWc21ot1Ys7GNEuVXOihGza-5XSL_$) = id;
  js.src = "[https://app.termly.io/embed-policy.min.js](https://urldefense.com/v3/__https://app.termly.io/embed-policy.min.js__;!!N3hqHg43uw!vlzinluv-33KHvobGc7SklLFAa05KN4mKVs23dcof7l5d7kRg7osS3JGx-kCTZe64JHCWc21ot1Ys7GNEuVXOihGza7wc5H4$)";
  tjs.parentNode.insertBefore(js, tjs);
}(document, 'script', 'termly-jssdk'));</script>
```

**ToU**

URL: [https://app.termly.io/document/terms-of-use-for-website/76a94ec4-766a-4a6a-b7fe-c68834af6811 ](https://urldefense.com/v3/__https://app.termly.io/document/terms-of-use-for-website/76a94ec4-766a-4a6a-b7fe-c68834af6811__;!!N3hqHg43uw!vlzinluv-33KHvobGc7SklLFAa05KN4mKVs23dcof7l5d7kRg7osS3JGx-kCTZe64JHCWc21ot1Ys7GNEuVXOihGzfa-ZmBY$) 
DIV:
```
<div name="termly-embed" data-id="76a94ec4-766a-4a6a-b7fe-c68834af6811" data-type="iframe"></div>
<script type="text/javascript">(function(d, s, id) {
  var js, tjs = d.getElementsByTagName(s)[0];
  if (d.getElementById(id)) return;
  js = d.createElement(s); [js.id](https://urldefense.com/v3/__http://js.id__;!!N3hqHg43uw!vlzinluv-33KHvobGc7SklLFAa05KN4mKVs23dcof7l5d7kRg7osS3JGx-kCTZe64JHCWc21ot1Ys7GNEuVXOihGza-5XSL_$) = id;
  js.src = "[https://app.termly.io/embed-policy.min.js](https://urldefense.com/v3/__https://app.termly.io/embed-policy.min.js__;!!N3hqHg43uw!vlzinluv-33KHvobGc7SklLFAa05KN4mKVs23dcof7l5d7kRg7osS3JGx-kCTZe64JHCWc21ot1Ys7GNEuVXOihGza7wc5H4$)";
  tjs.parentNode.insertBefore(js, tjs);
}(document, 'script', 'termly-jssdk'));</script>
```

**Comment by maherpa2** [comment]
@nhrishi Dan would be a good contact person for this. I recall that he did some work in this area so this would be a good starting point.

**Comment by nhrishi** [comment]
> @nhrishi Dan would be a good contact person for this. I recall that he did some work in this area so this would be a good starting point.

yes, I will discuss this with Dan today.

**Comment by rite2bala** [comment]
Please refer to Dan's input from an earlier email on 4th November 2020 -

He proposes:
"Help PharmaLedger improve its products and services by allowing analytics of usage data from your device.   You can change your decision later in Settings. All data is collected using privacy preserving techniques such as differential privacy and is not associated with you or your account"


cc: @nhrishi @maherpa2

**Comment by nhrishi** [comment]
Dear James,
 
A bit of a deeper dive:
 
My understanding is the intention is to use these for ePI—I have checked the content with the view to ask myself if the content is correct/applicable, are we consistent between the two docs and finally if we commit to something how can we actually fulfil the commitment.
It might make sense to have a call and align on next steps.
 
Both documents------PharmaLedger is spelled incorrectly---the l is lower case—the L needs to be capitalized
 
Privacy Policy
---The blanket statement “ we automatically collect device information, operating system ...etc----” is a bit of a question for me------even if the end users gives us permission we are not even able to collect all stated information---we need to discuss
 

 
---what changes are we expecting them to notify us with and what will we do with the information???

--not aligned with the Terms of Use


 
---we need to know what data and have a process to delete the data after 90 days?? what do we concretely do to take care of what we have committed to??
 

---how do we do this??what do we concretely do to take care of what we have committed to??
 

 
---we do not have any accounts---we need to play this out—what do we have in place should they contact us---what do we concretely do to take care of what we have committed to??
 

 
[---dpo@pharmaledger.health](mailto:---dpo@pharmaledger.health) account needs to be created
 
---I would want to validate this via a test case---

--California—residents---I want to have this as a case we walk through--
--updates—align with wording of Terms of Use


 
 
Terms of Use
 
 
---Thanks for the clarity re the word “Site”---then a suggestion I would remove the word website from the link and substitute it with site--- URL: [https://app.termly.io/document/terms-of-use-for-website/76a94ec4-766a-4a6a-b7fe-c68834af6811 ](https://urldefense.com/v3/__https:/app.termly.io/document/terms-of-use-for-website/76a94ec4-766a-4a6a-b7fe-c68834af6811__;!!N3hqHg43uw!vlzinluv-33KHvobGc7SklLFAa05KN4mKVs23dcof7l5d7kRg7osS3JGx-kCTZe64JHCWc21ot1Ys7GNEuVXOihGzfa-ZmBY$) ---- URL: [https://app.termly.io/document/terms-of-use-for-site/76a94ec4-766a-4a6a-b7fe-c68834af6811 ](https://app.termly.io/document/terms-of-use-for-site/76a94ec4-766a-4a6a-b7fe-c68834af6811) 
 
-I feel that the first and second sentence below contradict each other—in addition, my understanding is that the first sentence is accurate---there is currently not any place where the users can “contribute”—unless we consider the app stores and leaving messages there----same for the statement listed further down in the text----or am I missing something?
--- ----
 
 

 
 
---How does this sentence tie to our statements that all is open source—as we keep hearing anyone can download and make modifications etc....??

 
 
---First question is what would be “an applicable warranty”---secondly we do not expect in any situation a price to purchase to be applicable—so why are we including this??
---- 
Two points—Currently the text only includes Ireland---this is not aligned with the statement that you made below “For the moment I have included Ireland, Switzerland, Belgium and the US as placeholders”—nor is it aligned with the Privacy Policy--Secondly,  the I will add this to the configuration information discussion—that we can track in which country will the PRODUCTION site be hosted---or do we need to take into consideration where the demo site is hosted as well????
--- 
 
--if we intend to be able to invoke this scenario—we need to already add that to the backlog—know how we will do that and test it actually---it is a requirement then..

 
---we should be clear which data we are speaking about here---should be clearly defined and tested

 
[---contact@pharmaledger.health](mailto:---contact@pharmaledger.health) email address needs to be created
 
Kind regards,
Darlene
